ppx/ppx_js/tests/ppx.mlt

```diff
@@ -104,7 +104,7 @@ fun (obj : < > Js.t) -> obj##m
 
 [%%expect
 {|
-Line _, characters 24-27:
```

Comment:
This seems to be a regression. `obj`  has type `<  >` not `obj##m`

Reply:
Fixed.